component-library/components/sections/Left-right-block--Featured-map/Left-right-block--Featured-map.bookshop.yml

```diff
@@ -0,0 +1,80 @@
+# Metadata about this component, to be used in the CMS
+spec:
+  structures:
+    - content_blocks
+  label: "Contact block"
+  description: Contact content block
+  icon: contact_phone
+  tags: ['contact', 'content-block', 'left-right']
+
+# Defines the structure of this component, as well as the default values
+blueprint:
+  content:
+    social_icons: true
+    heading: 
+      primary_heading: A contact block
```

Comment:
I feel that this could maybe become a flat structure here? Feels weird for the editor to have to click in to be able to see one field

Reply:
Done